src/Components/InputTodo.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { runToast } from './toast/Toast';
+
+const InputTodo = ({ list, setList }) => {
+    const [form, setForm] = useState('');
+    const handleChange = (e) => {
```

Comment:
이벤트 핸들러의 이름은 보다 명확하게 `handle + 대상 + 동작` 형식으로 짓는 게 일반적이라고 합니다

```suggestion
    const handleInputChange = (e) => {
```

이런 식으로요~